docs/src/main/asciidoc/README.adoc

```diff
@@ -0,0 +1,238 @@
+:repo_url: https://github.com/marcingrzejszczak/database-update
+
+= Zero Downtime Deployment Database Issues
+
+{repo_url}
+
+This article will explain in more depth how to tackle issues related to database compatibility with Spring Boot and
```

Comment:
You don't need to mention Spring Boot this early. It's a fairly general topic, using a specific example with Spring Boot (which you could explain at the end of this paragraph, for instance).